service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/azure/database/AzureDatabaseUtilsRunner.java

```diff
@@ -0,0 +1,382 @@
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import java.util.UUID;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Component;
+
+/**
+ * Wrapper around <a href="{@docRoot}/azureDatabaseUtils/README.md">AzureDatabaseUtils</a>. Each
+ * public method in this class is a wrapper around a command in AzureDatabaseUtils. Every call
+ * creates a pod in the default namespace
+ */
+@Component
+public class AzureDatabaseUtilsRunner {
+  private static final Logger logger = LoggerFactory.getLogger(AzureDatabaseUtilsRunner.class);
+  public static final String POD_FAILED = "Failed";
+  public static final String POD_SUCCEEDED = "Succeeded";
+  // namespace where we create the pod to create the database
+  private static final String aksNamespace = "default";
+
+  private final AzureConfiguration azureConfig;
+  private final LandingZoneApiDispatch landingZoneApiDispatch;
+  private final WorkspaceService workspaceService;
+  private final SamService samService;
+  private final KubernetesClientProvider kubernetesClientProvider;
+
+  public AzureDatabaseUtilsRunner(
+      AzureConfiguration azureConfig,
+      LandingZoneApiDispatch landingZoneApiDispatch,
+      WorkspaceService workspaceService,
+      SamService samService,
+      KubernetesClientProvider kubernetesClientProvider) {
+    this.azureConfig = azureConfig;
+    this.landingZoneApiDispatch = landingZoneApiDispatch;
+    this.workspaceService = workspaceService;
+    this.samService = samService;
+    this.kubernetesClientProvider = kubernetesClientProvider;
+  }
+
+  /**
+   * Creates a database in the landing zone postgres server and grants the user access to it.
+   *
+   * @param azureCloudContext
+   * @param workspaceId
+   * @param podName
+   * @param userName
+   * @param userOid
+   * @param databaseName
+   * @throws InterruptedException
+   */
+  public void createDatabase(
+      AzureCloudContext azureCloudContext,
+      UUID workspaceId,
+      String podName,
+      String userName,
+      String userOid,
+      String databaseName)
+      throws InterruptedException {
+    final List<V1EnvVar> envVars =
+        List.of(
+            new V1EnvVar().name("spring_profiles_active").value("CreateDatabase"),
+            new V1EnvVar().name("NEW_DB_USER_NAME").value(userName),
+            new V1EnvVar().name("NEW_DB_USER_OID").value(userOid),
+            new V1EnvVar().name("NEW_DB_NAME").value(databaseName));
+    runAzureDatabaseUtils(
+        azureCloudContext,
+        workspaceId,
+        createPodDefinition(workspaceId, podName, envVars),
+        aksNamespace);
+  }
+
+  /**
+   * Creates a database in the landing zone postgres server and creates a role with access to it.
+   * The database role cannot be used to login to the database.
```

Comment:
The database role can read/write in the database, but can't log in to it? What role logs in? How/where is that created? Some additional comments would be helpful here.